api/src/main/java/io/strimzi/api/kafka/model/listener/arraylistener/GenericKafkaListenerConfiguration.java

```diff
@@ -168,6 +170,29 @@ public void setBrokers(List<GenericKafkaListenerConfigurationBroker> brokers) {
         this.brokers = brokers;
     }
 
+    @Description("The maximum number of connections we allow for this listener in the broker at any time. " +
+            "New connections are blocked if the limit is reached.")
```

Comment:
maybe "refused" or "dropped" sound better than "blocked"? But maybe a native English speaker opinion is more valuable than mine :-)

Reply:
`blocked` is what is used in the Kafka documentation: http://kafka.apache.org/documentation/#brokerconfigs_max.connections ... so I would stick with that.